### PR TITLE
Dependency version update

### LINKED ISSUE #513: Update the Out of date dependencies
* There are many major bug fixes and feature enhancements in the application dependencies

* Update the dependencies to the latest stable version to reduce this gap.

### PR DESCRIPTION
**Pull request by hemantjadon**
* Angular version bump.

* Angular material version bump and bug fixes

* CLI version update (Fixes major issues)

**Closes #513**

### CODE REVIEW

File: src/app/feed/info-box/info-box.component.ts [hunk 1]
```diff
@@ -12,17 +12,17 @@ import { Query } from '../../models';
 	styleUrls: ['./info-box.component.scss']
 })
 export class InfoBoxComponent implements OnInit, OnChanges {
-	@Input() private query: Query;
-	@Input() private apiResponseAggregations: ApiResponseAggregations;
-	private inviewporttwitters: Observable<boolean>;
-	private inviewportmentions: Observable<boolean>;
+	@Input() public query: Query;
```
Comment (singhpratyush) [inline src/app/feed/info-box/info-box.component.ts]: Is this to support the latest version?

File: src/app/media-wall/media-wall-custom-background/media-wall-custom-background.component.spec.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ describe('MediaWallCustomCardComponent', () => {
 	beforeEach(() => {
 		TestBed.configureTestingModule({
 			imports: [,
-				MdSlideToggleModule.forRoot(),
+				MdSlideToggleModule,
```
Comment (singhpratyush) [inline src/app/media-wall/media-wall-custom-background/media-wall-custom-background.component.spec.ts]: Are these changes to support latest version?

Comment (hemantjadon) [inline src/app/media-wall/media-wall-custom-background/media-wall-custom-background.component.spec.ts]: Yes the `forRoot` method is now deprecated.

File: src/app/feed/info-box/info-box.component.ts (hunk 1 above)
Comment (hemantjadon) [inline src/app/feed/info-box/info-box.component.ts]: Yes this is to support the latest version.

**Comment by mariobehling** [comment]
Good job!

**Pull request merged**